ds/data/processed/industry/employment_high_tech.nuts2.yaml

```diff
@@ -1,8 +1,8 @@
 api_doc_url: https://www.nomisweb.co.uk/api/v01/help
 api_type: POST
-data_date: 20210601
-subtitle: Employment in high tech industries based on the Business Register Employment Survey
+data_date: 20210623
 title: Employment in high tech industries
+subtitle: Employment in high tech industries based on the Business Register Employment Survey
 description: This indicator measures employment in high tech industries based on the Business Register Employment Survey. High tech industries are defined in terms of SIC-4 (industry) codes, using the definition in "The Geography of the UK's creative and high-tech economies" (https://media.nesta.org.uk/documents/geography_uks_creative_high-tech_economieswv20151.pdf) in table 6.1.3.4.
 endpoint_url: http://www.nomisweb.co.uk/api/v01/dataset/
 framework_group: place_potential
```

Comment:
(1) I think we need to add a warning here too, as the spec year seems to always be 2013.